server/tables/tables.go

```diff
@@ -569,6 +583,79 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
 			return nil
 		})
 	}
+
+	if m.HasTable("Invocations") && m.HasColumn(&Invocation{}, "invocation_uuid") {
+		if db.Dialector.Name() == mysqlDialect {
+			if err := db.Exec(`UPDATE Invocations SET invocation_uuid = UUID_TO_BIN(invocation_id) WHERE invocation_uuid IS NULL`).Error; err != nil {
+				return postMigrate, err
+			}
+			if err := db.Exec(
+				`UPDATE TargetStatuses ts
+			     SET invocation_uuid=(
+				   SELECT invocation_uuid FROM Invocations i 
+				   WHERE i.invocation_pk=ts.invocation_pk)
+				 WHERE invocation_uuid IS NULL  
+			    `).Error; err != nil {
+				return postMigrate, err
```

Comment:
nit: can we move these migrations into separate functions? this function is getting quite long and there's a lot of nesting

Reply:
Done.